filebeat/module/kibana/audit/ingest/pipeline-json.yml

```diff
@@ -0,0 +1,88 @@
+description: Pipeline for parsing Kibana audit logs in JSON format
+processors:
+- json:
```

Comment:
Would it be possible to move the json processor to the filebeat? The reason for this is that it makes it easier for users to have all fields available if they want to use any conditional filtering on the beats side.

In config.yml you can add for example:
```
processors:
  - decode_json_fields:
      fields: [message]
      target: json
```

Reply:
Good idea, done in [`e35d4d6` (#22696)](https://github.com/elastic/beats/pull/22696/commits/e35d4d68638a05ace19fb905b65538d72e2d502e)